Code/GraphMol/FileParsers/MolFileParser.cpp

```diff
@@ -2686,6 +2652,9 @@ RWMol *MolDataStreamToMol(std::istream *inStream, unsigned int &line,
   }
 
   if (res) {
+    res->clearAllAtomBookmarks();
```

Comment:
I don't think it is required, but I though it might be a good idea to clean up the bookmarks after they have been used to parse SGroups, so that these are "clean" after importing the molecule.